frontend/src/pages/SubmitPunchlinePage/index.tsx

```diff
@@ -141,7 +141,7 @@ const SubmitPunchlinePage = () => {
               handleOnClick={() => {
                 setPunchlineSubmitted(punchlineSelected);
                 setPunchlineSelected("");
-                memoryHistory.push("/postRound");
+                memoryHistory.push("/selectPunchline");
```

Comment:
Update the path in GameRouter to reflect the name change here

Reply:
Nevermind I thought you were renaming /postRound to /selectPunchline but you were just changing where it moved to hahah